server/requirements.txt

```diff
@@ -12,7 +12,7 @@ flask-restful>=0.3.9
 flask-sqlalchemy
 gunicorn
 humanize
-pquisby==0.0.17
+pquisby
```

Comment:
Are we sure that we want to let this "float"?

Given recent experiences, I think we should peg this at a "known good" version.